proprietary/tokens/src/brand/amsterdam/space.tokens.json

```diff
@@ -0,0 +1,11 @@
+{
+  "amsterdam": {
+    "space": {
```

Comment:
Do we want to use these tokens in Grid already as well? Or do that in a later PR?